node/src/components/small_network.rs

```diff
@@ -950,7 +955,7 @@ async fn server_task<P, REv>(
                 //       The code in its current state will consume 100% CPU if local resource
                 //       exhaustion happens, as no distinction is made and no delay introduced.
                 Err(err) => {
-                    warn!(%err, "{}: dropping incoming connection during accept", cloned_our_id)
+                    warn!(%cloned_our_id, %err, "dropping incoming connection during accept")
```

Comment:
```suggestion
                    warn!(our_id=%cloned_our_id, %err, "dropping incoming connection during accept")
```

Reply:
Done in [`af4ffe3` (#738)](https://github.com/CasperLabs/casper-node/pull/738/commits/af4ffe3d128db7e6ca26bdb73da6cc90cfb79a27)